plugins/BEdita/API/src/Controller/LoginController.php

```diff
@@ -422,14 +423,14 @@ public function change()
         $this->request->allowMethod(['patch', 'post']);
 
         if ($this->request->is('post')) {
-            $action = new ChangeCredentialsRequestAction();
+            $action = $this->createAction('ChangeCredentialsRequestAction');
             $action($this->request->getData());
 
             return $this->response
                 ->withStatus(204);
         }
 
-        $action = new ChangeCredentialsAction();
+        $action = $this->createAction('ChangeCredentialsRequestAction');
```

Comment:
maybe `$this->createAction('ChangeCredentialsAction');` ?

Reply:
already fixed ;)